charts/nfs-server-provisioner/templates/statefulset.yaml

```diff
@@ -117,6 +117,9 @@ spec:
       tolerations:
         {{- toYaml . | nindent 8 }}
       {{- end }}
+      {{- with .Values.priorityClass.name }}
+      priorityClassName: {{ . | quote }}
+      {{- end }}
```

Comment:
```suggestion
      {{- if .Values.priorityClass.create }}
      priorityClassName: {{ .Values.priorityClass.name | default (include "nfs-provisioner.fullname" .) | quote}}
      {{- end }}
```

Reply:
what about the use case where the user wants to re-use an existing priorityClass?

Reply:
Ah nice catch!

```suggestion
      {{- if (or .Values.priorityClass.name .Values.priorityClass.create)  }}
      priorityClassName: {{ .Values.priorityClass.name | default (include "nfs-provisioner.fullname" .) | quote }}
      {{- end }}
```

Reply:
> what about the use case where the user wants to re-use an existing priorityClass?

This makes sense, let's revert it then :)